FrontEnd/scripts/works.js

```diff
@@ -57,4 +71,26 @@ async function displayWork() {
   });
 }
 
+// NOTE: First display of the works
 displayWork();
+
+// NOTE: add listener for the diferent filters
+const buttonTous = document.querySelector(".btnTous");
+buttonTous.addEventListener("click", () => {
+  displayWork();
+});
+
+const buttonObject = document.querySelector(".btnObjets");
+buttonObject.addEventListener("click", () => {
+  displayWork("Objets");
+});
+
+const buttonAppartements = document.querySelector(".btnAppartements");
+buttonAppartements.addEventListener("click", () => {
+  displayWork("Appartements");
+});
+
+const buttonHotelRestaurant = document.querySelector(".btnHotelRestaurant");
+buttonHotelRestaurant.addEventListener("click", () => {
+  displayWork("Hotels & restaurants");
+});
```

Comment:
I think the next is to use the `/categories` endpoint to loop over all categories and generate addEventListener dynamically; you should at least add `TODO` comment explaining this.